src/apps/experimental/components/library/SortButton.tsx

```diff
@@ -16,7 +16,16 @@ import { LibraryTab } from 'types/libraryTab';
 import { ItemSortBy } from '@jellyfin/sdk/lib/models/api/item-sort-by';
 import { SortOrder } from '@jellyfin/sdk/lib/generated-client';
 
-const sortMenuOptions = [
+type SortOption = {
+    label: string;
+    value: ItemSortBy;
+};
+
+type SortOptionsMapping = {
+    [key: string]: SortOption[];
+};
```

Comment:
Could a `Record` type be used for this?

```suggestion
type SortOptionsMapping = Record<string, SortOption[]>;
```